src/cockpit/389-console/src/lib/database/localPwp.jsx

```diff
@@ -2380,290 +2514,325 @@ export class LocalPwPolicy extends React.Component {
+                                        </FormSelect>
+                                    </GridItem>
+                                </Grid>
+                                <Grid
+                                    title="Indicates the number of seconds that must pass before a user can change their password again. (passwordMinAge)."
+                                >
+                                    <GridItem className="ds-label" span={3}>
+                                        Password Minimum Age
+                                    </GridItem>
+                                    <GridItem span={9}>
+                                        <TextInput
+                                            value={this.state.passwordminage}
+                                            type="number"
+                                            id="passwordminage"
+                                            aria-describedby="horizontal-form-name-helper"
+                                            name="passwordminage"
+                                            onChange={(checked, e) => {
+                                                this.handleGeneralChange(e);
+                                            }}
+                                        />
+                                    </GridItem>
+                                </Grid>
+                                <Grid
+                                    title="The DN for a password administrator or administrator group (passwordAdminDN)."
+                                >
+                                    <GridItem className="ds-label" span={3}>
+                                        Password Administrator
+                                    </GridItem>
+                                    <GridItem span={9}>
+                                        <TextInput
+                                            value={this.state.passwordadmindn}
+                                            type="text"
+                                            id="passwordadmindn"
+                                            aria-describedby="horizontal-form-name-helper"
+                                            name="passwordadmindn"
+                                            onChange={(checked, e) => {
+                                                this.handleGeneralChange(e);
+                                            }}
+                                        />
+                                    </GridItem>
+                                </Grid>
+                            </Form>
+                            <Button
+                                isDisabled={this.state.saveGeneralDisabled}
+                                variant="primary"
+                                className="ds-margin-top-xlg ds-margin-left-sm"
+                                onClick={this.saveGeneral}
```

Comment:
I am getting an error when thying to edit the Local Password Policy.

```
CMD: saveGeneral: Saving general pwpolicy settings ==> dsconf -j ldapi://%2fvar%2frun%2fslapd-localhost.socket localpwp set dc=example,dc=com undefined=on
cockpit.js:1 usage: dsconf [-h] [-v] [-D BINDDN] [-w BINDPW] [-W] [-y PWDFILE] [-b BASEDN]
              [-Z] [-j]
              instance
              {backend,backup,chaining,config,directory_manager,monitor,plugin,pwpolicy,localpwp,replication,repl-agmt,repl-winsync-agmt,repl-tasks,sasl,security,schema,repl-conflict}
              ...
dsconf: error: unrecognized arguments: undefined=on

p @ cockpit.js:1
cockpit/$03530381300…sole/index.html#/:1 Uncaught SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at Function.<anonymous> (index.js:76)
    at cockpit.js:1
    at cockpit.js:1
    at A (cockpit.js:1)CMD: saveGeneral: Saving general pwpolicy settings ==> dsconf -j ldapi://%2fvar%2frun%2fslapd-localhost.socket localpwp set dc=example,dc=com undefined=on
cockpit.js:1 usage: dsconf [-h] [-v] [-D BINDDN] [-w BINDPW] [-W] [-y PWDFILE] [-b BASEDN]
              [-Z] [-j]
              instance
              {backend,backup,chaining,config,directory_manager,monitor,plugin,pwpolicy,localpwp,replication,repl-agmt,repl-winsync-agmt,repl-tasks,sasl,security,schema,repl-conflict}
              ...
dsconf: error: unrecognized arguments: undefined=on

p @ cockpit.js:1
cockpit/$03530381300…sole/index.html#/:1 Uncaught SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at Function.<anonymous> (index.js:76)
    at cockpit.js:1
    at cockpit.js:1
    at A (cockpit.js:1)
 ```

Reply:
There were settings in the local policies that only existed in the global policy.  So I had to remove a few settings.  Now it works